### DISCUSSION TITLE
cannot join slack

### DISCUSSION THREAD

**Discussion by GMW247** [libgit2/libgit2#6522] [Q&A] [answered]
the slack link at  http://slack.libgit2.org/ does not work

![Screenshot 2023-03-02 082124](https://user-images.githubusercontent.com/16270026/222455613-90a660cd-6284-4689-9072-fcdb400f6e2e.png)

this image is showing what error is returned.

i just wanted some help and libera is also quiet to getting help.

**Comment by ethomson** [comment, marked as answer]
Thanks for the report -- this should be fixed now!

**Reply by Some1and2-XC** [reply]
I seems like there is a regression. Doesn't work again :/

**Reply by ethomson** [reply]
Oops, this time it was DNS. Should be fixed now!
